boskos/client/client.go

```diff
@@ -93,6 +94,31 @@ func (c *Client) Acquire(rtype, state, dest string) (*common.Resource, error) {
 	return r, nil
 }
 
+// AcquireWait blocks until Acquire returns the specified resource or the
+// provided context is cancelled or its deadline exceeded.
+func (c *Client) AcquireWait(ctx context.Context, rtype, state, dest string) (*common.Resource, error) {
```

Comment:
hummm am I misreading? or I don't see how the deadline is configured?

Reply:
Hi @krzyzacy,

Deadline is part of the golang [`Context`](https://golang.org/pkg/context/). If a caller wants to wait two minutes, they would call:

```golang
AcquireWait(
	context.WithTimeout(context.Background(), time.Minute * 3), 
	"type", "state", "dest")`
```

Reply:
oh right :+1: I completely forgot that